src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php

```diff
@@ -93,7 +93,9 @@ public function configure(Event $event = null)
         }
         if (!$this->exceptionHandler) {
             if ($event instanceof KernelEvent) {
-                $this->exceptionHandler = array($event->getKernel(), 'terminateWithException');
+                if (is_callable(array($event->getKernel(), 'terminateWithException'))) {
```

Comment:
and ``is_callable`` would return true as soon there is ``__call`` even though the implementation does not support it, so it is not a silver-bullet either.